website/server/controllers/api-v3/tasks.js

```diff
@@ -578,11 +765,15 @@ api.scoreCheckListItem = {
  * @apiParam {String} taskId The task _id or alias
  * @apiParam {UUID} itemId The checklist item _id
  *
+ * @apiParam (body) {string} text The text that will replace the current checkitem text.
+ *
+ * @apiParamExample {json} Example body:
+ * {"text":"Czech 1"}
+ *
  * @apiSuccess {Object} data The updated task
  *
  * @apiUse TaskNotFound
  * @apiUse ChecklistNotFound
- * @apiUse ChallengeNotFound
```

Comment:
It should never happen but it could in case the challenge was removed from the database but its task were not